single-line-diagram/single-line-diagram-core/src/test/java/com/powsybl/sld/iidm/Test3WTPhaseTapChanger.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.sld.iidm;
+
+import com.powsybl.iidm.network.PhaseTapChanger;
+import com.powsybl.iidm.network.ThreeWindingsTransformer;
+import com.powsybl.iidm.network.ThreeWindingsTransformer.Leg;
+import com.powsybl.iidm.network.test.ThreeWindingsTransformerNetworkFactory;
+import com.powsybl.sld.builders.NetworkGraphBuilder;
+import com.powsybl.sld.model.graphs.VoltageLevelGraph;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author Giovanni Ferrari {@literal <giovanni.ferrari at soft.it>}
+ */
+class Test3WTPhaseTapChanger extends AbstractTestCaseIidm {
+
+    @BeforeEach
+    public void setUp() {
+        network = ThreeWindingsTransformerNetworkFactory.create();
+        network.getThreeWindingsTransformer("3WT").getLeg1().getTerminal().setP(-2800.0).setQ(400.0);
+        network.getThreeWindingsTransformer("3WT").getLeg2().getTerminal().setP(1400.0).setQ(400.0);
+        network.getThreeWindingsTransformer("3WT").getLeg3().getTerminal().setP(1400.0).setQ(400.0);
+        graphBuilder = new NetworkGraphBuilder(network);
+    }
+
+    @Test
+    void testVoltageLevelGraph3wtWithPhaseTagChanger() {
+        ThreeWindingsTransformer twt = network.getThreeWindingsTransformer("3WT");
+
+        addPhaseTapChanger(twt.getLeg1());
+        testSvg("/Test3WTPhaseTapChangerVoltageLevel1Leg.svg");
```

Comment:
Could you separate the testSvg in several tests? to have a single testSvg per `@Test`. It makes it easier when debugging